src/viewer.js

```diff
@@ -3193,6 +3194,8 @@ function onCanvasScroll( event ) {
             return false;   // We are swallowing this event
         }
     }
+
+    return true;
```

Comment:
I wonder if this should be an explicit `undefined`, since presumably that's what most MouseTracker handlers return as their "it's OK to propagate" value. What do you think?

Reply:
A agree, and had a feeling you'd ask! I'll fix, thanks!